doc/source/whatsnew/v1.3.0.rst

```diff
@@ -45,6 +45,8 @@ Other enhancements
 - Improved consistency of error message when passing an invalid ``win_type`` argument in :class:`Window` (:issue:`15969`)
 - :func:`pandas.read_sql_query` now accepts a ``dtype`` argument to cast the columnar data from the SQL database based on user input (:issue:`10285`)
 - Improved integer type mapping from pandas to SQLAlchemy when using :meth:`DataFrame.to_sql` (:issue:`35076`)
+- :func:`pandas.read_excel` can now auto detect .xlsb files (:issue:`35416`)
+- :func:`pandas.read_excel` now respects ``pandas.set_option`` (:issue:`34242`)
```

Comment:
not sure what this is in reference, can you clarify? (that is an unrelated issue).

Reply:
I've closed #38667 as a duplicate of #35416, the OP should be updated to link to the latter instead.

Reply:
@rhshadrach That issue should remain open until this is merged since the test still needs to be updated.

Reply:
@jreback, these are two somewhat related issues as they both pertain to engine=None. I figured I would solve both of them here. (Respecting set_option is a one-liner)

Reply:
#34242 refers to `NaT+Period doesnt match Period+NaT`, needs to be fixed. For #38667, it is a duplicate of #35416. #35416 will remain open until this is merged.

Reply:
Oh i meant #34252, I'll fix this.